src/common/services/Solana/GetData/ReadTokenWallet.tsx

```diff
@@ -0,0 +1,55 @@
+import {
```

Comment:
Should this not be generated by a cloud function?

Reply:
When a client verifies their public key the server can set these values in the User collection and then the client can fetch them as needed.

Reply:
Didn t we say we scan in the front then in the back if the front doesn t match the back ?

Reply:
let s just do it in the front for the mvp, especially when we are taking radical turns every other day